core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryRestApplication.java

```diff
@@ -94,9 +79,14 @@ public void setupResources(Configurable<?> config, SchemaRegistryConfig schemaRe
     config.register(new SubjectVersionsResource(schemaRegistry));
     config.register(new CompatibilityResource(schemaRegistry));
 
-    if (schemaRegistryResourceExtension != null) {
+    if (schemaRegistryResourceExtensions != null) {
       try {
-        schemaRegistryResourceExtension.register(config, schemaRegistryConfig, schemaRegistry);
+        for (SchemaRegistryResourceExtension
+            schemaRegistryResourceExtension : schemaRegistryResourceExtensions) {
+          if (schemaRegistryResourceExtension != null) {
```

Comment:
do we see a reason for this null check? the `getConfiguredInstances` should throw an exception if it is not able to instantiate an object.

Reply:
You can add a validator to protect against bad input.